### PR TITLE
Adding enets icon

### PR DESCRIPTION
**Pull request by ainzin**
** Checklist **
- [✓] All icons have a corresponding entry in `db/payment_icons.yml`
- [✓ ] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [✓] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [✓] I am confident that all icons are clear and easy to read/understand

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: app/assets/images/payment_icons/enets.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...3 3.166-.003 6.332-.002 9.498 0 .212 0 .421.019.63.03z"/><path fill="#0B4AB6" d="M15.495 9.831l4.587.003c.229 0 .459.014.666-.115a.262.262 0 0 1 .027.109c-.025.038-.053.076-.078.113-1.735.007-3.471.015-5.207.021l.005-.131z"/><path fill="#E0001D" d="M23.727 10.069c.174.089.143.254.143.402.002 1.489.002 2.979.002 4.468 0 .117-.004.23-.1.316-.15-.869-.045-1.744-.062-2.617-.017-.855.009-1.713.017-2.569zM9.872 15.254c-.158-.182-.087-.402-.088-.605-.006-1.375-.004-2.752-.003-4.128 0-.034.009-.079.031-.099.018-.015.063.001.096.004.011 1.609.082 3.22-.036 4.828zM14.069 8.796c.16.157.098.359.099.542.005 1.105.005 2.21 0 3.316 0 .18.069.388-.13.523l.031-4.381zM30.342 9.937c.725-.23 1.471-.084 2.203-.091.459-.004.949-.167 1.385.125l-3.588-.034zM30.455 12.598c.779-.243 1.578-.066 2.367-.1.287-.013.611-.129.871.119l-3.238-.019zM20.697 9.94c.025-.038.053-.076.078-.113.777.001 1.555 0 2.334.007.182.001.385-.065.529.118l-2.941-.012zM25.145 9.963c.195-.235.465-.137.695-.124.729.045 1.471-.149 2.188.111l-2.883.013zM29.771 9.988c.016-.074.035-.133.107-.035a.135.135 0 0 1-.107.035zM34.033 8.572c-.043.062-.08.056-.113-.007l.113.007zM24.996 10.318v4.701h-.057v-4.691a.964.964 0 0 0 .057-.01z"/></g></svg>
```
Comment (NathanPJF) [inline app/assets/images/payment_icons/enets.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-enets"><title id="pi-enets">eNETS</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#FFF" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><g fill-rule="evenodd" clip-rule="evenodd"><path fill="#E41132" d="M20.691 8.541l.172.01c.162.202.391.124.592.124 3.881.004 7.764.004 11.645 0 .201 0 .43.08.592-.124l.229.014c.033.063.07.069.113.007 1.213.121 1.881.85 1.867 2.022-.002.327-.17.474-.461.449-.346-.029-.852.201-.938-.375-.059-.384-.246-.576-.572-.698-.436-.292-.926-.129-1.385-.125-.732.007-1.479-.139-2.203.091l-.463.016c-.072-.099-.092-.039-.107.035-.482.478-.402.933.227 1.095.305.079.582.19.953.165 1.125-.074 2.27-.138 3.391.092.789.162 1.387.598 1.551 1.434.172.869.088 1.689-.643 2.318a1.482 1.482 0 0 1-.998.383c-1.328-.002-2.656.031-3.98-.008-1.391-.043-2.211-.891-2.146-2.098.014-.277.016-.457.369-.334a.616.616 0 0 0 .398-.002c.426-.172.604-.012.633.414.025.355.271.502.584.523 1.285.084 2.574.064 3.861.012.326-.014.529-.184.553-.541.023-.354-.096-.599-.447-.705-.129-.038-.256-.08-.383-.12-.26-.248-.584-.131-.871-.119-.789.034-1.588-.143-2.367.1-1.773-.014-2.393-.691-2.428-2.646-.718-.26-1.461-.065-2.189-.11-.23-.014-.5-.112-.695.124-.004 1.593-.008 3.186-.016 4.779-.002.26.072.617-.264.68-.359.068-.787.25-1.094-.166.096-.086.1-.199.1-.316 0-1.488 0-2.979-.002-4.468 0-.148.031-.313-.143-.402l-.088-.117c-.145-.183-.348-.116-.529-.118-.779-.008-1.557-.006-2.334-.007a.278.278 0 0 0-.027-.109c.033-.397.098-.793-.057-1.179zm4.305 1.777l-.057.01v4.691h.057v-4.701z"/><path fill="#E40E30" d="M15.489 12.727l.001 1.281c-.135.475.166 1.02-.232 1.453-.872.154-1.49-.107-2.02-.889-.901-1.328-1.933-2.569-2.921-3.837-.101-.129-.132-.396-.409-.309-.033-.003-.078-.019-.096-.004-.022.02-.031.064-.031.099-.001 1.375-.003 2.753.003 4.128.001.203-.069.424.088.605-.311.412-.745.238-1.103.166-.373-.076-.256-.465-.257-.738-.006-1.824.013-3.651-.014-5.476-.007-.505.195-.656.657-.706.851-.092 1.463.117 1.973.898.774 1.188 1.693 2.281 2.556 3.411.103.135.235.246.354.369.199-.136.129-.344.13-.523.006-1.106.006-2.211 0-3.316-.001-.184.061-.385-.099-.542.34-.433.803-.265 1.233-.25.302.393.079.864.193 1.285l-.005.131-.001 1.29v1.474z"/><path fill="#0B4AB6" d="M4.759 10.265c.525-.024 1.046.156 1.501.505.26.199.55.442.281.813-.294.404-.57.17-.872-.051-.642-.469-1.444-.452-2.036.004a1.692 1.692 0 0 0-.543 1.968c.258.662.968 1.104 1.696 1.049.378-.027.7-.184.983-.432.25-.217.514-.277.759-.006.278.307.051.521-.164.729-.76.738-2.061.845-3.054.256a2.573 2.573 0 0 1-1.179-2.93c.327-1.165 1.346-1.92 2.628-1.905zM15.258 15.461c.398-.434.097-.979.232-1.453 1.575 0 3.15.029 4.723-.014.672-.018.557.385.572.783.016.418.023.734-.576.719-1.649-.043-3.3-.027-4.951-.035zM15.489 12.727v-1.475c1.497 0 2.994.029 4.49-.01.609-.015.631.295.631.75-.002.452-.02.765-.631.75-1.496-.041-2.993-.015-4.49-.015z"/><path fill="#0046B7" d="M15.495 9.831c-.114-.421.109-.892-.193-1.285 1.797-.054 3.593-.045 5.389-.005.154.386.09.783.057 1.178-.207.129-.438.115-.666.115l-4.587-.003z"/><path fill="#0B4AB6" d="M4.989 13.313l-1.264.001c-.281 0-.417-.145-.435-.42-.018-.277.113-.475.376-.481a52.68 52.68 0 0 1 2.754.002c.256.008.388.235.358.502-.026.237-.163.389-.413.392-.459.007-.918.002-1.377.002l.001.002z"/><path fill="#E0001D" d="M33.691 8.551c-.162.203-.391.124-.592.124-3.881.004-7.764.004-11.645 0-.201 0-.43.079-.592-.124.9-.011 1.801-.029 2.701-.03 3.166-.003 6.332-.002 9.498 0 .212 0 .421.019.63.03z"/><path fill="#0B4AB6" d="M15.495 9.831l4.587.003c.229 0 .459.014.666-.115a.262.262 0 0 1 .027.109c-.025.038-.053.076-.078.113-1.735.007-3.471.015-5.207.021l.005-.131z"/><path fill="#E0001D" d="M23.727 10.069c.174.089.143.254.143.402.002 1.489.002 2.979.002 4.468 0 .117-.004.23-.1.316-.15-.869-.045-1.744-.062-2.617-.017-.855.009-1.713.017-2.569zM9.872 15.254c-.158-.182-.087-.402-.088-.605-.006-1.375-.004-2.752-.003-4.128 0-.034.009-.079.031-.099.018-.015.063.001.096.004.011 1.609.082 3.22-.036 4.828zM14.069 8.796c.16.157.098.359.099.542.005 1.105.005 2.21 0 3.316 0 .18.069.388-.13.523l.031-4.381zM30.342 9.937c.725-.23 1.471-.084 2.203-.091.459-.004.949-.167 1.385.125l-3.588-.034zM30.455 12.598c.779-.243 1.578-.066 2.367-.1.287-.013.611-.129.871.119l-3.238-.019zM20.697 9.94c.025-.038.053-.076.078-.113.777.001 1.555 0 2.334.007.182.001.385-.065.529.118l-2.941-.012zM25.145 9.963c.195-.235.465-.137.695-.124.729.045 1.471-.149 2.188.111l-2.883.013zM29.771 9.988c.016-.074.035-.133.107-.035a.135.135 0 0 1-.107.035zM34.033 8.572c-.043.062-.08.056-.113-.007l.113.007zM24.996 10.318v4.701h-.057v-4.691a.964.964 0 0 0 .057-.01z"/></g></svg>
```

Comment (NathanPJF) [reply app/assets/images/payment_icons/enets.svg]: The `width` and `height` attributes are provided twice. This suggestion removes the duplicates.

**Comment by ainzin** [comment]
Thanks Nathan!

**Comment by ainzin** [comment]
Hi there,

Couldn't find any eNETS icon in payment types. Please advise. 

Thanks!

**Comment by NathanPJF** [comment]
If you mean the SVG, the file is here: https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/enets.svg

If that's not what you meant, could you please expand on what you were looking for?

**Comment by NathanPJF** [comment]
Update: I just saw your PR https://github.com/activemerchant/payment_icons/pull/155

I'll work on a new release now :)

**Pull request merged**